src/translations/en.json

```diff
@@ -407,6 +407,44 @@
       }
     },
     "dialogs": {
+      "quick_open": {
+        "commands": {
+          "heading": "YAML configuration reloading",
+          "introduction": "Some parts of Home Assistant can reload without requiring a restart. Hitting reload will unload their current YAML configuration and load the new one.",
```

Comment:
This should not be de heading/introduction for commands as they will probably not stay limited to reloading YAML

Reply:
Yep. Already removed locally. I'll push up what I have so far.

Reply:
Done

Reply:
Done